rad/src/operators/array.rs

```diff
@@ -155,29 +165,49 @@ pub fn filter(
             };
             let subscript = unpack_subscript(first_arg).map_err(subscript_err)?;
 
-            let mut result = vec![];
+            let mut reports = vec![];
+            let mut results = vec![];
+
+            let settings = RadonScriptExecutionSettings::enable_all();
```

Comment:
This should depend on the context, right? If partial results are disabled they should remain disabled, no need to always accumulate the partial reports.

Reply:
True. I should start tagging my TODOs as such so that I don't forget to change stuff once I get them working :sweat_smile:

Reply:
Done. Commit 5c4d20e01284b600e41f7b2b404747381bb23419 adds a factory that creates instances of `RadonScriptExecutionSettings` that are appropriate for each specific `Stage`.

Reply:
Partial results was an optional feature in the first place :sweat_smile: